src/doc/trpl/rust-inside-other-languages.md

```diff
@@ -336,6 +336,58 @@ print the result.
 
 On my system, this takes a quick `0.092` seconds.
 
+# Go
+
+Go may not have a GIL, but there are cases where you may need to
+avoid its garbage collector.
+
+In order to do FFI in Go, we first need to download the library:
+
+```bash
+$ go get bitbucket.org/binet/go-ffi/pkg/ffi
+```
+
+(You may need to install mercurial first).
+
+After it's installed, we can use ffi:
+
+```go
+package main                                                                                                                                                                                                         
+
+import "bitbucket.org/binet/go-ffi/pkg/ffi"                                                                                                                                                                          
+import "fmt"                                                                                                                                                                                                         
+
+func main() {                                                                                                                                                                                                        
+    lib, _ := ffi.NewLibrary("target/release/libembed.so")                                                                                                                                                           
+    process, _ := lib.Fct("process", ffi.Void, []ffi.Type{})                                                                                                                                                         
+    process()                                                                                                                                                                                                        
+    fmt.Println("done!")                                                                                                                                                                                             
+}
+```
+
+This also looks a great deal like the Ruby and Node examples.
+We use the `ffi` module obtained from
+bitbucket.org/binet/go-ffi/pkg/ffi to get `ffi.NewLibrary()`, which
```

Comment:
So that URI isn't actually an HTTP URL: it's the go get-able URL. If you go to it with a web browser it breaks.

I can certainly link to https://bitbucket.org/binet/go-ffi/ in the markdown instead of repeating the go-gettable URL however.